src/integration.rs

```diff
@@ -1638,6 +1645,8 @@ impl<A: AllocatorTrait> Integration<A> {
         }
         .expect("Failed to create descriptor pool.");
 
+        println!("Create new pool");
```

Comment:
Maybe change this to a log message.

Reply:
Yup, i changed it into an `info!` log from [7539ea3](https://github.com/MatchaChoco010/egui-winit-ash-integration/pull/13/commits/7539ea3b1df8a17ae6e5210175e049fb1d1e4484) onwards